src/components/attendee/BoothGrid/BoothCard.module.scss

```diff
@@ -0,0 +1,31 @@
+@use "scss/attendee/border";
+@use "scss/attendee/font";
+@use "scss/attendee/space";
+@use "scss/attendee/effects/sparkle";
+@use "scss/attendee/effects/rounded";
+@use "scss/attendee/effects/atom";
+@use "scss/attendee/theme";
+@use "scss/attendee/breakpoint";
+
+$booth-width: 255px;
+$booth-height: 255px;
+
+.container {
+  @include sparkle.panel--glass;
+  @include rounded.normal;
+  width: $booth-width;
+  height: $booth-height;
+  overflow: hidden;
+}
+
+.contents {
+  padding: space.gap(6) space.gap(12);
+}
+
+.title {
+  display: block;
+  margin-bottom: space.gap(12);
+  @include font.xlarge;
+  border-bottom: 1px solid rgba(130, 130, 130, 0.3);
```

Comment:
let's use `border.$width--slim` instead of this 1px and replace rgba with `theme.$sparkle--border`